apm-commons/apm-util/src/main/java/org/apache/skywalking/apm/util/PropertyPlaceholderHelper.java

```diff
@@ -16,27 +16,24 @@
  *
  */
 
-package org.apache.skywalking.oap.server.library.util;
+package org.apache.skywalking.apm.util;
 
-import com.google.common.base.Strings;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Properties;
 import java.util.Set;
-import org.apache.logging.log4j.core.util.Assert;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import java.util.logging.Logger;
 
 /**
  * Utility class for working with Strings that have placeholder values in them. A placeholder takes the form {@code
  * ${name}}. Using {@code PropertyPlaceholderHelper} these placeholders can be substituted for user-supplied values. <p>
  * Values for substitution can be supplied using a {@link Properties} instance or using a {@link PlaceholderResolver}.
  */
 public class PropertyPlaceholderHelper {
-    private static final Logger logger = LoggerFactory.getLogger(PropertyPlaceholderHelper.class);
+    private static final Logger logger = Logger.getLogger(PropertyPlaceholderHelper.class.getName());
```

Comment:
This log makes me a little awkward. Logger is definitely different in agent and collector. By using `java.util.logging.Logger` will make the log into application log, right? But I also haven't better idea.

Reply:
Create a bridge like log4j instead of logback.

Reply:
Maybe a light bridge mechanism? Make `PropertyPlaceholderHelper` abstract, and ask each side implements a `log(String` method?

Reply:
It doesn't matter. I can delete the logger for simplicity. :D

Reply:
If this log isn't so important, and we could know the value in log from outside of `PropertyPlaceholderHelper`. Delete is fine.